var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -141,4 +163,19 @@ def install(self, spec, prefix):
         install_tree('lib', prefix.lib)
         install_tree('include', prefix.include)
         if '~mpi' in spec:
-            install('libseq/libmpiseq.a', prefix.lib)
+            lib_dsuffix = '.dylib' if sys.platform == 'darwin' else '.so'
+            lib_suffix = lib_dsuffix if '+shared' in spec else '.a'
+            install('libseq/libmpiseq%s' % lib_suffix, prefix.lib)
+
+        if '~mpi' in spec: # FIXME: A process or daemon was unable to complete a TCP connection to another process
+            with working_dir('examples', create=False):
+                mpirun = join_path(self.spec['mpi'].prefix.bin, 'mpirun') # FIXME: avoid hardcoding mpirun (could be mpiexec, mpiexec.hydra, etc)
```

Comment:
i would gladly test this part, but i had some strange issues with MPI running tests: `A process or daemon was unable to complete a TCP connection to another process`, which I could not make sense of.